database/benches/hash_keys.rs

```diff
@@ -48,13 +48,13 @@ struct Address([u8; 20]);
 
 impl Distribution<Blake2bHash> for Standard {
     fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> Blake2bHash {
-        Blake2bHash(rng.gen())
+        Blake2bHash(rng.r#gen())
```

Comment:
Probably better to change to rand 0.9, where this is called `rng.random()`.

Reply:
I started doing that on branch `rand-0.9` based on `albatross`, but there are some trait changes that I don't understand. If you could look at the compiler errors, that would be great! :pray: